Gemfile.lock

```diff
@@ -24,8 +24,8 @@ GEM
       safe_yaml (~> 1.0.0)
     diff-lcs (1.2.5)
     docile (1.1.5)
-    hashie (3.5.5)
```

Comment:
This commit shouldn't touch dependencies and shouldn't include any Gemfile.lock changes.

Reply:
👍